app/components/preprint-status-banner/component.js

```diff
@@ -224,14 +226,12 @@ export default Component.extend({
     }),
 
     decisionChanged: computed('submission.reviewsState', 'decision', function() {
-        return this.get('submission.reviewsState') !== this.get('decision');
+        return ((this.get('submission.reviewsState') !== this.get('decision') && this.get('submission.reviewsState') !== PENDING) ||
+        (this.get('submission.reviewsState') === PENDING && this.get('decision') !== ACCEPTED));
```

Comment:
Stop me if I'm getting too picky, but this is getting hard to read, and I'd like to avoid state-dependent logic when we can -- it might be cleaner to save an `initialDecision` somewhere (similar to `initialReviewerComment`), and check `decision !== initialDecision`